### PR TITLE
Rename properties to new terms

### PR DESCRIPTION
**Pull request by jlurien**
- Applied lowerCamelCase for property names, UpperCamelCase for schema names
- Updated API_documentation as well. This is a redundant effort that hopefully we don't have to maintain in the future, as being discussed in https://github.com/camaraproject/WorkingGroups/issues/164
- In a separate PR we should enhance descriptions, once Glossary definitions are consolidated.

### CODE REVIEW

**Comment by hdamker** [comment]
We should wait with this PR until https://github.com/camaraproject/WorkingGroups/issues/157 is finally decided with a vote as requested by Telefonica.

**Review by hdamker — APPROVED** [review]

**Comment by hdamker** [comment]
Guess we are ready to due these changes now as the guidelines are also updated.

@eric-murray @jlurien @patrice-conil @sfnuser @akoshunyadi would you have a look and approve as well?

File: code/API_definitions/qod-api.yaml [hunk 1]
```diff
@@ -63,41 +63,41 @@ paths:
                     code: INVALID_ARGUMENT
                     message: "Schema validation failed at  ..."
                 MsisdnRequired:
```
Comment (sfnuser) [inline code/API_definitions/qod-api.yaml:65]: Should we update this line too?

File: documentation/API_documentation/QoD_API.md [hunk 2]
```diff
@@ -26,14 +26,14 @@ Security access keys such as OAuth 2.0 client credentials used by client applica
 **QoS profiles and QoS profile labels**
 Latency or throughput requirements of the application mapped to relevant QoS profile class.
 
-**Identifier for the the user equipment (UE)**
-At least one identifier for the user equipment out of four options: IPv4 address, IPv6 address, MSISDN, or external ID [[5]](#5) assigned by the mobile network operator for the user equipment
+**Identifier for the the device**
```
Comment (akoshunyadi) [inline documentation/API_documentation/QoD_API.md:29]: typo: the the

File: documentation/API_documentation/QoD_API.md [hunk 3]
```diff
@@ -26,14 +26,14 @@ Security access keys such as OAuth 2.0 client credentials used by client applica
...
+At least one identifier for the device (user equipment) out of four options: IPv4 address, IPv6 address, Phone number, or Network Access Identifier [[5]](#5) assigned by the mobile network operator for the device.
 
 **Identifier for the application server (AS)**
```
Comment (akoshunyadi) [inline documentation/API_documentation/QoD_API.md:32]: AS is not used anymore, more occurrences are affected

File: code/API_definitions/qod-api.yaml (hunk 1 above)
Comment (hdamker) [reply code/API_definitions/qod-api.yaml:65]: I would say so.

**Comment by hdamker** [comment]
@jlurien will resolve the comment or should I?

File: code/API_definitions/qod-api.yaml (hunk 1 above)
Comment (hdamker) [reply code/API_definitions/qod-api.yaml]: Done in https://github.com/camaraproject/QualityOnDemand/pull/129/commits/ed3bbdc954ff21982ceee48b840c6d44d8aa9dec

File: documentation/API_documentation/QoD_API.md (hunk 2 above)
Comment (hdamker) [reply documentation/API_documentation/QoD_API.md]: Done in https://github.com/camaraproject/QualityOnDemand/pull/129/commits/fe2cef99b05112008c4d5f59316cbfb4e1b1dd85

File: documentation/API_documentation/QoD_API.md (hunk 3 above)
Comment (hdamker) [reply documentation/API_documentation/QoD_API.md]: Done in https://github.com/camaraproject/QualityOnDemand/pull/129/commits/fe2cef99b05112008c4d5f59316cbfb4e1b1dd85 (plus replaced one remaining occurence of UE).

**Comment by hdamker** [comment]
AS @jlurien seems not be available I have addressed the comments above.
@akoshunyadi and @sfnuser can you shortly check and approve?

**Comment by sfnuser** [comment]
> AS @jlurien seems not be available I have addressed the comments above. @akoshunyadi and @sfnuser can you shortly check and approve?

@hdamker - Cheers. Are we planning to update the commit in two other places (line 83 & 95) as well?

**Comment by hdamker** [comment]
@sfnuser :

> Are we planning to update the commit in two other places (line 83 & 95) as well?

Thanks for the hint ... have done further updates of error examples in lines 71, 77, 83, 89, 95 to get all the error examples consistent with the new property terms. Please have a view.

**Comment by sfnuser** [comment]
> @sfnuser :
> 
> > Are we planning to update the commit in two other places (line 83 & 95) as well?
> 
> Thanks for the hint ... have done further updates of error examples in lines 71, 77, 83, 89, 95 to get all the error examples consistent with the new property terms. Please have a view.

Thanks. Looks good to me.

**Review by sfnuser — APPROVED** [review]

**Review by hdamker — APPROVED** [review]
Hopefully we have now found all occurence which need to be replaced.

**Review by akoshunyadi — APPROVED** [review]

**Pull request merged**